src/resources/views/base/inc/head.blade.php

```diff
@@ -11,7 +11,11 @@
 
     @if (config('backpack.base.styles') && count(config('backpack.base.styles')))
         @foreach (config('backpack.base.styles') as $path)
-        <link rel="stylesheet" type="text/css" href="{{ asset($path).'?v='.config('backpack.base.cachebusting_string') }}">
+            @if(is_array($path))
```

Comment:
When you're echoing multiple bassets one after another, the output is on the same line, which looks odd:

![CleanShot 2023-01-03 at 21 16 26](https://user-images.githubusercontent.com/1032474/210425814-34524070-7aec-4f08-9401-7302bd0e5de2.png)

Instead of making the source code look super-clean, it does the opposite.